src/lib/InstantSearch.ts

```diff
@@ -285,6 +288,21 @@ See ${createDocumentationLink({
     return this;
   }
 
+  /**
+   * Removes a middleware from the InstantSearch lifecycle.
+   */
+  public unuse(...middlewareToUnuse: Middleware[]): this {
+    this.middleware
+      .filter(m => middlewareToUnuse.includes(m.middleware))
```

Comment:
use `indexOf` here, `includes` doesn't exist on ie11

Reply:
Yeah but we already use `includes` in InstantSearch.js and addressed in the guide: https://www.algolia.com/doc/guides/building-search-ui/installation/js/#browser-support

Reply:
I didn't realise we required it to be polyfilled now, lgtm!